docs: remove duplicate text (#6077)
aws/eks-anywhere: docs/content/en/docs/getting-started/docker/_index.md

   ```

   You can now use the cluster like you would any Kubernetes cluster.

> 1. To deploy a test application run the following command.

Reviewer: ```suggestion
1. The following command will deploy a test application:
```
